akka-http/src/main/scala/akka/http/javadsl/server/directives/FramedEntityStreamingDirectives.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.http.javadsl.server.directives
+
+import java.util.function.{ Function ⇒ JFunction }
+import java.util.{ List ⇒ JList, Map ⇒ JMap }
+
+import akka.NotUsed
+import akka.http.javadsl.marshalling.Marshaller
+import akka.http.javadsl.model.{ HttpEntity, _ }
+import akka.http.javadsl.server.{ EntityStreamingSupport, Route }
+import akka.http.javadsl.unmarshalling.Unmarshaller
+import akka.http.scaladsl.marshalling.{ Marshalling, ToResponseMarshallable }
+import akka.http.scaladsl.server.{ Directives ⇒ D }
+import akka.stream.javadsl.Source
+import akka.util.ByteString
+
+/** EXPERIMENTAL API */
+abstract class FramedEntityStreamingDirectives extends TimeoutDirectives {
+
+  import akka.http.javadsl.server.RoutingJavaMapping._
+  import akka.http.javadsl.server.RoutingJavaMapping.Implicits._
+
+  @CorrespondsTo("asSourceOf")
+  def entityAsSourceOf[T](um: Unmarshaller[ByteString, T], support: EntityStreamingSupport,
```

Comment:
That's the `javadsl.unmarshallling` one, so not the one that the alias point to.